### PR TITLE
UI Backlog fixes

### PR DESCRIPTION
**Pull request by rmonroe**

### CODE REVIEW

File: teamwork/templates/projects/view_project.html [hunk 1]
```diff
@@ -137,63 +137,85 @@
...
+                            <a href="{% url 'profile' project.ta.username %}">
+                                <span class="label label-info">
+                                    {{ project.ta.username }}
+                                </span>
+                            </a>
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <strong>Members</strong>
+                            &nbsp;
+                            {% for mem in project.members.all %}
+                            {% if mem.username != null %}
```
Comment (Kapn) [inline teamwork/templates/projects/view_project.html:183]: Do we need this if statement?

File: teamwork/templates/projects/view_project.html [hunk 2]
```diff
@@ -739,7 +761,7 @@ <h3 class="box-title">Team Chat</h3>
     <div class="box-header with-border">
         <h6 class="box-title">Pending Requests to Join: ( {{ pending_count }} )</h6>
         <div class="box-tools">
-            <button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-minus"></i></button>
+            <button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-plus"></i></button>
```
Comment (Kapn) [inline teamwork/templates/projects/view_project.html:764]: Noice

File: teamwork/templates/projects/view_project.html [hunk 3]
```diff
@@ -137,63 +137,85 @@
...
+                            <a href="{% url 'profile' project.scrum_master.username %}">
+                                <span class="label label-info">
+                                    {{ project.scrum_master.username }}
+                                </span>
+                            </a>
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <strong>Meeting Location</strong>
+                            &nbsp;
+                            {% if project.ta_location == "" %}
+                                N/A
```
Comment (Kapn) [inline teamwork/templates/projects/view_project.html:204]: should we put these N/A in an html tag

Comment (rmonroe) [reply teamwork/templates/projects/view_project.html:204]: They're just filler text if the field is empty, they are fine being out

File: teamwork/templates/projects/view_project.html (hunk 2 above)
Comment (rmonroe) [reply teamwork/templates/projects/view_project.html:764]: Lol you started with a minus for some reason when the box was already closed

File: teamwork/templates/projects/view_project.html (hunk 1 above)
Comment (rmonroe) [reply teamwork/templates/projects/view_project.html:183]: Why not its just a little safety net

**Pull request merged**